### DISCUSSION TITLE
Changes not showing in the website

### DISCUSSION THREAD

**Discussion by shahanas5s** [community/community#54840] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Hi, I made a portfolio website in github. And I edited the index.html directly in github and clicked commit changes button. But even  after 2 days, the change is not reflecting in the website

**Comment by iamreiyn** [comment, marked as answer]
Hi @shahanas5s 

The time it takes for the CDN cache to clear may vary depending on how your domain is [set up](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site). Using a subdomain like yourwebsite.github.io is recommended, but it may take longer for page builds to appear due to the GitHub CDN.

Press **Ctrl + Shift + R** on Windows/Linux within your website to perform a refresh that clears a cache for that particular page, or **try clearing the cache data of your browser**. If you would like to try other ways, consider checking [this](https://stackoverflow.com/questions/24851824/how-long-does-it-take-for-github-page-to-show-changes-after-changing-index-html) Stack Overflow question.

Hope it helps! If I was able to help you, kindly consider *Marking this as Answer* so people with the same issue can find the appropriate answer. If you need additional help, feel free to let me know! Thank you.
